src/System.Security.Cryptography.Cng/ref/project.json

```diff
@@ -1,8 +1,8 @@
 {
   "dependencies": {
-    "System.Runtime": "4.0.0",
-    "System.IO": "4.0.0",
-    "System.Runtime.Handles": "4.0.0",
+    "System.Runtime": "4.1.0-rc3-24117-00",
```

Comment:
@bartonjs @ericstj - We should make sure that this version bump is actually what we want here. The other option is to down-grade the version these packages in the dependent Algorithms (and Encoding) crypto packages.